### PR TITLE
Fixes broken curl and parametrize python and conda versions

### PR DESCRIPTION
**Pull request by brenoarosa**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,24 +11,32 @@ LABEL org.label-schema.name="Apache PySpark $SPARK_VERSION" \
...
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -Ls https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh --output /tmp/miniconda.sh && \
     bash /tmp/miniconda.sh -b -f -p "/opt/miniconda3" && \
-    rm /tmp/miniconda.sh && \
-    conda config --set auto_update_conda true && \
-    conda config --set channel_priority false && \
+    rm /tmp/miniconda.sh
+
+ENV PATH="/opt/miniconda3/bin:${PATH}"
```
Comment (kenjihiraoka) [inline Dockerfile:29]: what do you think about removing the `ENV` from here and let it together with others at the beginning of the dockerfile?

**Review by kenjihiraoka — APPROVED** [review]
lgtm, just a comment about a pattern to let the ENV together

File: Dockerfile (hunk 1 above)
Comment (brenoarosa) [reply Dockerfile:29]: I'm not sure where it would be better placed since the path doesn't exist during the first steps.

Comment (brenoarosa) [reply Dockerfile:29]: I'll merge despite this comment as it is a minor issue.

**Pull request merged**